zds/utils/management/commands/remove_one_year_old_ip_addresses.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime, timedelta
+
+from django.core.management.base import BaseCommand
+
+from zds.utils.models import Comment
+
+
+class Command(BaseCommand):
+    help = "Removes IP addresses that are more than one year old"
+
+    def handle(self, *args, **options):
+        one_year_ago = datetime.now() - timedelta(days=365)
+        Comment.objects.filter(pubdate__lte=one_year_ago).update(ip_address="")
```

Comment:
```suggestion
        Comment.objects.filter(pubdate__lte=one_year_ago,~Q(ip_address="")).update(ip_address="")
```
Parce que sinon ça va tout le temps faire une requête `UPDATE` énorme, alors qu'il n'y a pas besoin de traiter à nouveau les commentaires plus vieux d'un an qui ont déjà été traité la veille.